.circleci/config.yml

```diff
@@ -909,14 +909,14 @@ jobs:
           path: /tmp/test-results
       - slack/status:
           fail_only: true
-          failure_message: "Acceptance tests against Kind with Kubernetes v1.23 with Consul 1.11.x nightly failed. Check the logs at: ${CIRCLE_BUILD_URL}"
+          failure_message: "Acceptance tests against Kind with Kubernetes v1.23 with Consul 1.11 nightly failed. Check the logs at: ${CIRCLE_BUILD_URL}"
 
   acceptance-kind-1-23-consul-nightly-1-12:
     environment:
       - TEST_RESULTS: /tmp/test-results
       - CONSUL_IMAGE: "docker.mirror.hashicorp.services/hashicorppreview/consul-enterprise:1.12-dev"
       - ENVOY_IMAGE: "envoyproxy/envoy:v1.22.2"
-      - CONSUL_K8S_IMAGE: "docker.mirror.hashicorp.services/hashicorp/consul-k8s-control-plane:0.45.0"
+      - CONSUL_K8S_IMAGE: "docker.mirror.hashicorp.services/hashicorp/consul-k8s-control-plane:0.47.1"
```

Comment:
Does this need to be the default image or this specific image version?

Reply:
The intention was to run latest release against previous consul versions.

Reply:
If there are changes in the helm chart that require a dev image, then these tests will break, right? I think if we wanted to run the release image, we'd need to use a tag of the helm chart to make sure it always works.

Reply:
Yeah I think that makes sense!

Reply:
I've added a new flag for helm chart version which plumbs into the `testConfig.Version` param we already had in place, it defaults to the chart-dir.